### PR TITLE
Rnd 2154 csi update

### PR DESCRIPTION
**Pull request by TomasBankauskas**

### CODE REVIEW

**Comment by youvalv** [comment]
Task linked: [RND-2154 Update /create page themes to work with CSI](https://app.clickup.com/t/24537050/RND-2154)

File: src/utils/content.ts [hunk 1]
```diff
@@ -2,12 +2,11 @@ import * as fs from 'fs';
 import path from 'path';
 import glob from 'glob';
 import frontmatter from 'front-matter';
-import sbConfig from './../../stackbit.config';
 import * as types from 'types';
 
-if (!sbConfig.pagesDir || !sbConfig.dataDir) throw new Error('Invalid Stackbit config file');
-
-export const siteConfigFile = sbConfig.dataDir + '/config.json';
+export const dataDir = 'content/data';
```
Comment (eladroz) [inline src/utils/content.ts:7]: Now that Git CSI module doesn't have a distinction between contentDir & pagesDir, we'll need to change the code in this module & similar starters (but - this is outside the scope of this PR)

File: stackbit.config.ts [hunk 2]
```diff
@@ -10,34 +13,33 @@ import { Link } from './.stackbit/models/Link';
...
-    nodeVersion: '16',
-    dataDir: 'content/data',
-    pagesDir: 'content/pages',
-    pageLayoutKey: 'type',
-    styleObjectModelName: 'ThemeStyle',
-    assets: {
+const gitContentSource = new GitContentSource({
+    rootPath: __dirname,
+    contentDirs: ['content'],
+    models: [Button, Card, CardsSection, Config, Footer, Header, HeroSection, Image, Link, Page, ThemeStyle],
```
Comment (eladroz) [inline stackbit.config.ts:19]: Please put all models in an array in a separate file (as done in content-ops) so you can also remove all model imports here and replace them with a single one.

**Review by eladroz — CHANGES_REQUESTED** [review]

File: src/utils/content.ts (hunk 1 above)
Comment (TomasBankauskas) [reply src/utils/content.ts:7]: got it

File: stackbit.config.ts (hunk 2 above)
Comment (TomasBankauskas) [reply stackbit.config.ts]: updated, please check @eladroz.

**Review by eladroz — APPROVED** [review]
